### PR TITLE
Add documentation about shared memory

### PR DESCRIPTION
**Pull request by tobsan**
Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

**Review by kursatkobya — APPROVED** [review]

File: doc/chapters/integration-guidelines/10-shared-memory.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Working with shared memory in containers
+========================================
+
+There are two main types of shared memory in Linux systems: POSIX shm and SysV IPC. POSIX shm
+generally assumes that a there is a tmpfs mounted on ``/dev/shm`` where one can handle the shared
+memory objects, which are accessed through its own family of API calls. POSIX shm is not namespace
+aware.
+
+SysV IPC uses mechanisms built-in to the kernel (internally it uses a tmpfs as well, but it hidden).
```
Comment (frznlogic) [inline doc/chapters/integration-guidelines/10-shared-memory.rst]: but it hidden!

File: doc/chapters/integration-guidelines/10-shared-memory.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Among other things, SysV IPC supports shared memory, and is namespace aware.
```
Comment (frznlogic) [inline doc/chapters/integration-guidelines/10-shared-memory.rst]: namespace aware feels awkward, namespace-aware or aware of namespaces perhaps?

**Review by frznlogic — CHANGES_REQUESTED** [review]
Minor changes.

File: doc/chapters/integration-guidelines/10-shared-memory.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
+Working with shared memory in containers
+========================================
+
+There are two main types of shared memory in Linux systems: POSIX shm and SysV IPC. POSIX shm
+generally assumes that a there is a tmpfs mounted on ``/dev/shm`` where one can handle the shared
+memory objects, which are accessed through its own family of API calls. POSIX shm is not namespace
+aware.
```
Comment (frznlogic) [inline doc/chapters/integration-guidelines/10-shared-memory.rst]: Same problem here, sorry, didn't see it before.

**Review by frznlogic — APPROVED** [review]

**Review by thenor — APPROVED** [review]

**Pull request merged**